scripts/create_build_id.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+clean_tag_element() {
+    local tag_element="$1"
+    echo "${tag_element//\//-}"
+}
+
+generate_tag() {
+    local branch_name="$1"
+    local build_id="$2"
+    local git_hash="$3"
+
+    local clean_branch_name
```

Comment:
I think we don't need to declated `clean_branch_name` and `lean_build_id` separately. I think these declarations can be inlined with line 16 and 17 like below:
`
local clean_branch_name=$(clean_tag_element "$branch_name")
local clean_build_id=$(clean_tag_element "$build_id")
`